com.playeveryware.eos/Documentation~/google_signin.md

```diff
@@ -0,0 +1,54 @@
+<a href="/com.playeveryware.eos/README.md"><img src="/com.playeveryware.eos/Documentation~/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
+
+# <div align="center">Connect Login with GoogleID using Credential Manager</div>
+---
+
+## Overview
+
+This document describes how to enable Google ID authentication in conjunction with Epic Online Services Plugin for Unity.  
+We followed [Authenticate users with Sign in with Google](https://developer.android.com/identity/sign-in/credential-manager-siwg) and implemented a plugin for the users to easily achieve the feature.
+
+## Setup
+
+### Plugins
+The feature is included in our samples, so no external plugin is needed.
+
+### Build Settings
+To allow the feature, follow the build configuration step below. 
```

Comment:
I think I'd use the phrase "To enable the feature," instead of "To allow"

Reply:
That is the word I was looking for, thanks.